File: Day-45/Lavanya.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> ans;
        int m=n;
        for(int i=0;i<n;i++){
            ans.push_back(nums[i]);
            ans.push_back(nums[m]);
            m++;
        }
        return ans;
    }
};

File: Day-56/Lavanya.py
class Solution:
    def toGoatLatin(self, S: str) -> str:
        newList= S.split()
        ans=[]
        vowels = ['a','e','i','o','u','A','E','I','O','U']
        for i, word in enumerate(newList):
            if word[0] in vowels:
                word = word + 'ma'
            else:
                word = word[1:] + word[0] + 'ma'
            for j in range(i+1):
                word +='a'
            ans.append(word)
        newString = ' '.join(ans)
        return newString
